website/docs/getting-started/paths-mapping.md

```diff
@@ -76,6 +76,8 @@ const { compilerOptions } = require('./tsconfig')
 /** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
   // [...]
+  roots: ['<rootDir>'],
+  modulePaths: [compilerOptions.baseUrl], // <-- This will be set to './src'
```

Comment:
```suggestion
  modulePaths: [compilerOptions.baseUrl], // <-- This will be set to 'baseUrl' value
```

Reply:
Would you pls correct the same comment for other files?

Reply:
I'm sorry for having you check on this several times. The comments are now consistent across all versions.